src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -233,31 +164,41 @@ def setup(self, builder: Builder):
             / data_values.US_POPULATION
         )
 
-    ########################
-    # Event-driven methods #
-    ########################
+    def get_configuration_name(self) -> str:
+        return "_".join(self.name.split("_observer_"))
+
+    def register_observations(self, builder: Builder) -> None:
+        # While this is a concatenating observation, it is not registered as such
+        # because we need to modify the per-time-step results_gatherer method
+        # FIXME: Is tracked==True correct? We probably want to exclude households
+        # that are entirely untracked, but should we allow for simulants
+        # to, say, copy from household members that are now untracked?
```

Comment:
This is a good question. Im pretty sure the copy from household member looks at the household to see if there are members to copy so I would guess you can't copy an untracked simulant but testing that would be bettert han taking my word for it.